modules/pubmaticBidAdapter.js

```diff
@@ -71,25 +71,25 @@ function _parseAdSlot(bid) {
   bid.params.adSlot = _cleanSlot(bid.params.adSlot);
 
   var slot = bid.params.adSlot;
-  var splits = slot.split(':');
+  try{
+    var splits = slot.split('@');
 
-  slot = splits[0];
-  if (splits.length == 2) {
-    bid.params.adUnitIndex = splits[1];
-  }
-  splits = slot.split('@');
-  if (splits.length != 2) {
-    utils.logWarn('AdSlot Error: adSlot not in required format');
-    return;
+    slot = splits[0];
+    if (splits.length == 2) {
+      bid.params.adUnitIndex = splits[1].split(":")[1];
+    }
+    splits = splits[1].split(":")[0].split("x");
+    if (splits.length != 2) {
+      utils.logWarn('AdSlot Error: adSlot not in required format');
```

Comment:
Not specific to this changes but all messages should be of format:
utils.logWarn(BIDDER_CODE : Message)
Also it would be good to print the format here i.e <adunit>@<width>x<height>